src/DropdownMenu.tsx

```diff
@@ -0,0 +1,39 @@
+import React from 'react';
+import { Person } from './types/Person';
+
+interface Props {
+  people: Person[];
+  onSelected: (person: Person) => void;
+}
+
+export const DropdownMenu: React.FC<Props> = ({
+  people,
+  onSelected,
+}) => {
+  return (
+    <div className="dropdown-menu" role="menu">
+      <div className="dropdown-content">
+        {people.length > 0
```

Comment:
```suggestion
        {!people.length
```

Reply:
Probably !!people.length? But elsint  says that's not the way to do it. So I think it's better to keep it that way.